src/components/Navigation.astro

```diff
@@ -0,0 +1,29 @@
+<nav class="fixed flex justify-center w-screen h-10 bg-sky-500">
```

Comment:
this shouldn't be fixed (but you should keep fixed, just not on this element

Reply:
I didn't change it yet. Can you give me a hit?

Reply:
Sure. Just make the header fixed and make sure it sticks to the top and to the left

Reply:
The container class in existing navigation code has no effect as it's `fixed`. So we should keep the `<header>` fixed, put the `.container` inside and add it's content below.